controllers/funding/programmes/index.js

```diff
@@ -48,6 +48,21 @@ router.get('/', injectHeroImage('rosemount-1-letterbox-new'), async function (
             req.query.location
         );
 
+        const covidStatuses = get(response.meta, 'covid19Statuses');
+        if (covidStatuses) {
+            let countryCovidStatus;
+            countryCovidStatus = covidStatuses.find(
+                (status) => status.country === locationParam
+            );
+            // Fall back to the UK-wide statement
+            if (!countryCovidStatus) {
+                countryCovidStatus = covidStatuses.find(
+                    (status) => status.country === 'ukWide'
+                );
+            }
+            res.locals.countryCovidStatus = countryCovidStatus;
```

Comment:
You might be able to combine this logic like this:

```js
res.locals.countryCovidStatus = covidStatuses.find(function (status) {
    return (
        status.country === locationParam ||
        status.country === 'ukWide'
    );
});
```
Or, possibly:

```js
res.locals.countryCovidStatus = covidStatuses.find(function (
    status
) {
    if (status.country === locationParam) {
        return true;
    } else {
        return status.country === 'ukWide';
    }
});
```

Might not be following this logic though so ignore this if I've oversimplified things

Reply:
No this is a good shout, used the first approach 👍